locales/ace/app.po

```diff
@@ -2497,6 +2508,25 @@ msgctxt "travel_search_engine_fxlite"
 msgid "See results in %1$s Travel Discovery"
 msgstr ""
 
+#. this is a title indicates that users will see search results summerized by
```

Comment:
English is a bit broken in the comment: "This is a title indicating that … summarized".

More important: What is "Travel Discovery"? Is it an external brand name that shouldn't be translated? Is it the name of a feature? I see it was exposed as a normal text in older strings, so I would expect it to be translated at this point.

Reply:
Hey Flod,
Thanks for raising this! I'll rewrite the comment to make it more clear.

"Travel Discovery" is an important feature name that is present on multiple UIs and Play Store introduction. From both the UX and Marketing expectations, we hope it to be consistent and translated. 

We found some translations are using

- Multiple translation forms (https://pontoon.mozilla.org/id/firefox-rocket/app.po/?search=travel+discovery&string=206155 & https://pontoon.mozilla.org/id/firefox-rocket/app.po/?search=travel+discovery&string=207081 ) 
- Mixing local language and English (https://pontoon.mozilla.org/zh-TW/firefox-rocket/app.po/?search=travel+discovery&string=206155 & https://pontoon.mozilla.org/zh-TW/firefox-rocket/app.po/?search=travel+discovery&string=207081)

To meet the expectations mentioned above, we'd like to simply reserve the “Travel Discovery” as a non-translatable feature name.

Reply:
The end of the message says the opposite of the first part:
- `we hope it to be consistent and translated`
- `we'd like to simply reserve the “Travel Discovery” as a non-translatable feature name`

At this point I assume you want to keep it untranslated? In that case, using a variable is the right approach.

Even if you want to keep it translated but consistent, you can use the variable, and then expose "Travel Discovery" for translation only once.

Reply:
Yes, we want to keep it untranslated, so we can leverage the existing translation on Google Play Store and campaigns. 

For the locales that we haven't run any campaigns, we'll do our best to have them translated. Locales with very few users may see English, and we're well aware of that and fine to live with it.

Reply:
The way I understand it, you're going to translate this feature name outside of the normal localization file, i.e. directly in the code. That's a process that scales really poorly, but it's your choice to make.

P.S. CC @gueroJeff in case we need to explain this to community in the near future.